blueflood-core/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/RollupHandler.java

```diff
@@ -56,114 +63,145 @@
 
     private static final boolean ROLLUP_REPAIR = Configuration.getInstance().getBooleanProperty(CoreConfig.REPAIR_ROLLUPS_ON_READ);
     private ExecutorService ESUnitExecutor = null;
+    private ListeningExecutorService RollupsOnReadExecutor = null;
```

Comment:
nit: Having a private variable with uppercase name is a little weird and does not follow our conventions. 
Possibly ESUnitExecutor is ok because ES is an abbreviation.